src/extension.ts

```diff
@@ -48,46 +48,55 @@ export async function activate(context: vscode.ExtensionContext) {
     }
   }
 
-  const vitestCmd = getVitestCommand(
-    vscode.workspace.workspaceFolders[0].uri.fsPath,
-  ) ?? {
-    cmd: 'npx',
-    args: ['vitest'],
-  }
+  const vitestConfigs: {
+    workspace: vscode.WorkspaceFolder
+    cmd: string
+    args: string[]
+    version: string | null
+  }[] = await Promise.all(vscode.workspace.workspaceFolders.map(async (folder) => {
+    const cmd = getVitestCommand(folder.uri.fsPath)
 
-  const vitestVersion = await getVitestVersion(vitestCmd, getConfig().env || undefined).catch(async (e) => {
-    log.appendLine(e.toString())
-    log.appendLine(`process.env.PATH = ${process.env.PATH}`)
-    log.appendLine(`vitest.nodeEnv = ${JSON.stringify(getConfig().env)}`)
-    let errorMsg = e.toString()
-    if (!isNodeAvailable(getConfig().env || undefined)) {
-      log.appendLine('Cannot spawn node process')
-      errorMsg += 'Cannot spawn node process. Please try setting vitest.nodeEnv as {"PATH": "/path/to/node"} in your settings.'
-    }
+    const version = await getVitestVersion(cmd, getConfig().env || undefined).catch(async (e) => {
```

Comment:
`getConfig` should have a WorkspaceFolder param now, to use it in the workspace.getConfiguration

```ts
export function getConfiguration(section?: string, scope?: ConfigurationScope | null): WorkspaceConfiguration;
```

`!getConfig().enable` above need to be changed too